kubectl-rook-ceph.sh

```diff
@@ -44,6 +44,7 @@ function print_usage() {
   echo "    status                                  : print the phase and conditions of the CephCluster CR"
   echo "    status all                              : print the phase and conditions of all CRs"
   echo "    status <CR>                             : print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc"
+  echo "    purge-osd <osd-id> [--force]            : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs."
```

Comment:
Should this really be a subcommand of `rook`? What if we had this be a top-level command? @travisn thoughts?